dbt_project.yml

```diff
@@ -29,6 +29,10 @@ vars:
     hubspot_contact: "{{ source('rag_hubspot', 'contact') }}"
     hubspot_deal: "{{ source('rag_hubspot', 'deal') }}"
     hubspot_owner: "{{ source('rag_hubspot', 'owner') }}"
+
+    rag_using_zendesk: true
+    rag_using_jira: true
+    rag__using_hubspot: true
```

Comment:
These variables are inconsistent. Zendesk and Jira have a single underscore between rag and using; whereas hubspot has two.

Additionally, now that you added the default variable value for these within the macro in the `rag__unified_document` model, this call is no longer required. 

I would prefer we do not define these variables here and instead rely on the defaults within the models/macros themselves.
```suggestion
```